packages/devextreme/scss/widgets/fluent/htmlEditor/_sizes.scss

```diff
@@ -9,6 +9,8 @@ $fluent-html-editor-horizontal-padding: null !default;
 $fluent-html-editor-add-image-dialog-fileuploader-padding: null !default;
 $fluent-html-editor-toolbar-menu-separator-margin: 4px !default;
 $fluent-html-editor-fileuploader-input-wrapper-border-radius: 8px !default;
+$fluent-html-editor-fileuploader-input-wrapper-border-size: 1.5px !default;
+$fluent-html-editor-add-image-dialog-tabs-padding: 12px !default;
```

Comment:
Looks like that padding should be 14px, to align the tab text relative to the content. It is also necessary to consider the compact mode and put the correct value for it